src/hotspot/share/jvmci/jvmciCodeInstaller.cpp

```diff
@@ -854,7 +854,7 @@ JVMCI::CodeInstallResult CodeInstaller::install(JVMCICompiler* compiler,
 
 void CodeInstaller::initialize_fields(HotSpotCompiledCodeStream* stream, u1 code_flags, methodHandle& method, CodeBuffer& buffer, JVMCI_TRAPS) {
   if (!method.is_null()) {
-    _parameter_count = method->size_of_parameters();
+    _parameter_count = method->number_of_parameters();
```

Comment:
_parameter_count is used to create a OopMap, so it must be in the number of words that are ocupied by all the aeguments, not the number of parameters.

https://github.com/openjdk/jdk/blob/27a38d9093958ae4851bc61b8d3f0d71dc780823/src/hotspot/share/jvmci/jvmciCodeInstaller.cpp#L263